res/openvpn-gui-res-it.rc

```diff
@@ -332,127 +347,126 @@ Completa la richiesta di autorizzazione precedente."
     IDS_ERR_DUP_HANDLE_IN_WRITE "DuplicateHandle on hInputWrite fallito."
     IDS_ERR_CREATE_PROCESS "CreateProcess fallito, exe='%ls' cmdline='%ls' dir='%ls'"
     IDS_ERR_CREATE_THREAD_STATUS "CreateThread fallito per visualizzare lo stato."
-    IDS_NFO_STATE_WAIT_TERM "Stato corrente: In attesa che OpenVPN termini…"
-    IDS_NFO_STATE_CONNECTED "Stato corrente: Connesso"
+    IDS_NFO_STATE_WAIT_TERM "Stato attuale: In attesa che OpenVPN termini…"
+    IDS_NFO_STATE_CONNECTED "Stato attuale: Connesso"
     IDS_NFO_NOW_CONNECTED "%ls è ora connesso."
     IDS_NFO_ASSIGN_IP "IP assegnato: %ls"
-    IDS_ERR_CERT_EXPIRED "Non riesco a connettermi perché il tuo certificato è scaduto o l'orologio di sistema non è corretto."
-    IDS_ERR_CERT_NOT_YET_VALID "Non riesco a connettermi perché il tuo certificato non è ancora valido. Assicurati che l'orologio di sistema sia corretto."
-    IDS_NFO_STATE_RECONNECTING "Stato corrente: Riconnessione in corso"
-    IDS_NFO_STATE_DISCONNECTED "Stato corrente: Disconnesso"
+    IDS_ERR_CERT_EXPIRED "Impossibile connettersi perché il certificato è scaduto o l'orologio di sistema non è aggiornato."
```

Comment:
maybe "impostato correttamente" rather than "aggiornato"? Not sure it is common to say "aggiornarne l'orologio"